pkg/reconciler/volumeclaim/pvchandler.go

```diff
@@ -73,14 +80,20 @@ func (c *defaultPVCHandler) CreatePVCFromVolumeClaimTemplate(ctx context.Context
 			if apierrors.IsAlreadyExists(err) {
 				c.logger.Infof("Tried to create PersistentVolumeClaim %s in namespace %s, but it already exists",
 					claim.Name, claim.Namespace)
+			} else if apierrors.IsForbidden(err) {
+				if strings.Contains(err.Error(), "exceeded quota") {
+					// This is a retry-able error
+					return fmt.Errorf("%w: %v", ErrPvcCreationFailedRetryable, err.Error())
+				}
+				return fmt.Errorf("%w for %s: %w", ErrPvcCreationFailed, claim.Name, err)
```

Comment:
This return and the one in the `else` right after can be simplified.


```suggestion
			} else if apierrors.IsForbidden(err) && strings.Contains(err.Error(), "exceeded quota") {
       			// This is a retry-able error
				return fmt.Errorf("%w: %v", ErrPvcCreationFailedRetryable, err.Error())
			}
			return fmt.Errorf("%w for %s: %w", ErrPvcCreationFailed, claim.Name, err)

```